includes/admin/class-sensei-setup-wizard.php

```diff
@@ -142,6 +142,13 @@ public function remove_notices_from_setup_wizard( $current_screen ) {
 	 * @access private
 	 */
 	public function enqueue_scripts() {
+		/**
+		 * Triggered on the Setup Wizard page to add additional scripts and behaviors to the page.
+		 *
+		 * @since $$next-version$$
+		 * @hook sensei_setup_wizard_enqueue_scripts
+		 */
+		do_action( 'sensei_setup_wizard_enqueue_scripts' );
```

Comment:
Do we really need this for a special reason? Or could we just use the `admin_enqueue_scripts` in other scopes?

If you agree that we could use the `admin_enqueue_scripts`, I'd use this PR to change  the `prepare_wizard_page` to use the hook `admin_enqueue_scripts` instead of the `admin_print_scripts` and `admin_print_styles`.

Reply:
Hmmmm, good question, @renatho! 

I tried to look for a more general action to use instead of creating one, but I coulnd't find one. I think we could use `admin_enqueue_scripts` without any problems, but the main issue I see is that we'll still need some checking on Sensei Pro side to enqueue the script/add the action only when the page is correct.

With that said, I'll take a look at changing the code to use `admin_enqueue_scripts` instead. Thanks for the suggestion!

Reply:
> the main issue I see is that we'll still need some checking on Sensei Pro side to enqueue the script/add the action only when the page is correct.

I wouldn't see any problem with duplicating the check there, since it's simple. But if you don't like this for any reason, and prefer to keep the special hook here, I'm fine. 😉

Reply:
No worries, I'm just a little slow today, but I did the changes you proposed: [9778d64](https://github.com/Automattic/sensei/pull/5794/commits/9778d641c69cc2ed55fde26f8bd0cff8ad42141a) and [9a2cb3a](https://github.com/Automattic/sensei/pull/5794/commits/9a2cb3acb058fd355c0c7201e0052917c3d59d4c)

I also made the necessary changes in Sensei Pro. :)

Reply:
I update the PR to reflect the new change. :)

Reply:
Thanks!